packages/backend-function/src/factory.ts

```diff
@@ -276,10 +260,9 @@ const isWholeNumberBetweenInclusive = (
   max: number
 ) => min <= test && test <= max && test % 1 === 0;
 
-export type NodeVersion = 16 | 18 | 20;
+export type NodeVersion = 18 | 20;
```

Comment:
Are we ok with removing support for Node 16 before Lambda deprecates it? See https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Reply:
Technically we already don't support it because the secret fetcher shim uses aws sdk v3 (which is not available in node 16 runtimes). Personally, I'm okay with this since we are still in preview and anyone using this is setting up a new lambda so they should be using an LTS version

Reply:
I've updated to include a node 16 shim as well